src/scenes/NftDetailPage/NftDetailAsset.tsx

```diff
@@ -75,40 +78,58 @@ function NftDetailAsset({ nft }: Props) {
   const { aspectRatioType } = useContentState();
   const breakpoint = useBreakpoint();
 
+  const isCollectorsNoteEnabled = isFeatureEnabled(FeatureFlag.COLLECTORS_NOTE);
+
   // We do not want to enforce square aspect ratio for iframes https://github.com/gallery-so/gallery/pull/536
   const isIframe = getMediaType(nft) === NftMediaType.ANIMATION;
   const shouldEnforceSquareAspectRatio =
     breakpoint === size.desktop ||
     breakpoint === size.tablet ||
-    (aspectRatioType !== 'wide' && !isIframe);
+    (aspectRatioType === 'square' && !isIframe);
 
   return (
-    <StyledAssetContainer
-      maxHeight={maxHeight}
-      shouldEnforceSquareAspectRatio={shouldEnforceSquareAspectRatio}
-    >
-      <NftDetailAssetComponent nft={nft} maxHeight={maxHeight} />
-    </StyledAssetContainer>
+    <>
+      <StyledAssetContainer
+        footerHeight={GLOBAL_FOOTER_HEIGHT}
+        maxHeight={maxHeight}
+        shouldEnforceSquareAspectRatio={shouldEnforceSquareAspectRatio}
+        isCollectorsNoteEnabled={isCollectorsNoteEnabled}
+      >
+        <NftDetailAssetComponent nft={nft} maxHeight={maxHeight} />
+      </StyledAssetContainer>
+      {isCollectorsNoteEnabled && (userOwnsAsset || assetHasNote) && (
+        <NftDetailNote
+          nftId={nft.id}
+          userOwnsAsset={userOwnsAsset}
+          nftCollectorsNote={nft?.collectors_note}
+        />
+      )}
+    </>
   );
 }
 
 type AssetContainerProps = {
+  footerHeight: number;
   maxHeight: number;
   shouldEnforceSquareAspectRatio: boolean;
+  isCollectorsNoteEnabled: boolean;
 };
 
 const StyledAssetContainer = styled.div<AssetContainerProps>`
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: center;
+  position: relative;
+  z-index: 2; /* Above footer in event they overlap */
 
   ${({ shouldEnforceSquareAspectRatio }) =>
     shouldEnforceSquareAspectRatio ? 'aspect-ratio: 1' : ''};
 
-  @media only screen and ${breakpoints.desktop} {
-    width: ${({ maxHeight }) => maxHeight}px;
-    height: ${({ maxHeight }) => maxHeight}px;
+  @media only screen and ${breakpoints.tablet} {
+    width: 100%;
+    ${({ isCollectorsNoteEnabled, footerHeight }) =>
+      isCollectorsNoteEnabled ? `max-height: calc(85vh - 46px - ${footerHeight}px)` : ''};
```

Comment:
curious, where did the `46px` come from?

Reply:
This is roughly equal to the height of the "Collector's Note" title + the first line of text + padding. It was kind of some back-and-forth editing to find the magic number. 

On second thought, we might only want to subtract the 46px **if the asset has a note** or **if the user owns the current asset** (e.g. has the `Add Collector's Note` button).